### PR TITLE
Periphery outputs

### PR DESCRIPTION
**Pull request by fedealconada**
## Motivation

- [x] Update periphery address
- [x] Upload outputs
- [x] Polish Periphery and Factories tasks 
- [x] Bonus track: deployed 2 factories on Goerli (MockFactory and ERC4626Factory)  

## Implementer Checklist

- [ ]  [OPTIONAL] Create a reference implementation (in python or JS)
- [ ]  Create a short "Motivation" section for the PR
- [ ]  Write a spec for the feature and put it in the PR description – basic function names and expected state transitions is OK
- [ ]  Get the PR reviewed by at least two people

If this is your first time reviewing smart contract changes
- [ ]  Review the [Solcurity](https://github.com/Rari-Capital/solcurity) standard

If smart contract changes were made
- [ ]  Check all of the new revert paths with concrete tests
- [ ]  Check all of the new storage slot writes with concrete tests – do not make the passing test case the trivial case!
- [ ]  Go line-by-line through the new code and ensure it has all been covered by concrete tests – since we don’t have a coverage engine for foundry yet, you can imagine how one might check that each different code path is covered
- [ ]  Simplify the implementation and spend some time trying to minimize gas costs
- [ ]  Write fuzz tests for the feature – try everything to break the implementation (is this function monotonically in/decreasing, should it always be less than something else, etc)
- [ ]  Capture bugs discovered during the above steps in concrete tests (regression tests)
- [ ]  Add integration tests – fuzz or concrete tests that test how the new code affects the entire system, either locally or with a mainnet fork
- [ ]  [OPTIONAL] Integrate the feature into the deployment scripts
- [ ]  [OPTIONAL] Add new sanity checks to the deployment scripts

If your PR uses or interacts with prices of any kind
- [ ] Make sure forge is running the test on the latest block
- [ ] Compare the value returned with some other source (like Curve)
- [ ] Check the `updatedAt` (if available) value of the contract and confirm that the price we are sourcing is in fact not older than X
- [ ] Ensure oracle liveness from the oracle maintainer, either through a public status interface or through written confirmation from a developer.

### CODE REVIEW

**Review by jparklev — APPROVED** [review]
me parece bien (?)

> Bonus track

⏺️

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ cUSDC | [0x39aa39c021dfbae8fac545936693ac917d5e7563](https://etherscan.io/token/
 Contract | Address
 --------- | -------------
 [Divider](https://github.com/sense-finance/sense-v1/blob/dev/pkg/core/src/Divider.sol) | [0x86bA3E96Be68563E41c2f5769F1AF9fAf758e6E0](https://etherscan.io/address/0x86bA3E96Be68563E41c2f5769F1AF9fAf758e6E0#code)
-[Periphery](https://github.com/sense-finance/sense-v1/blob/dev/pkg/core/src/Periphery.sol) | [0xFff11417a58781D3C72083CB45EF54d79Cd02437](https://etherscan.io/address/0xFff11417a58781D3C72083CB45EF54d79Cd02437#code)
+[Periphery](https://github.com/sense-finance/sense-v1/blob/dev/pkg/core/src/Periphery.sol) | [0xaa17633AA5A3Cb56698838561161bdb16Cebb8E3autoRollerFactory](https://etherscan.io/address/0xaa17633AA5A3Cb56698838561161bdb16Cebb8E3autoRollerFactory#code)
```
Comment (jparklev) [inline README.md:33]:
```suggestion
[Periphery](https://github.com/sense-finance/sense-v1/blob/dev/pkg/core/src/Periphery.sol) | [0xaa17633AA5A3Cb56698838561161bdb16Cebb8E3](https://etherscan.io/address/0xaa17633AA5A3Cb56698838561161bdb16Cebb8E3#code)
```

File: pkg/deployments/tasks/20221031-factories-and-morpho/index.js [hunk 2]
```diff
@@ -82,85 +82,93 @@ task("20221031-factories-and-morpho", "Deploys 4626 Factories & maUSDC & maUSDT"
...
-          params: [senseAdminMultisigAddress],
-        });
-
-        const multisigSigner = await hre.ethers.getSigner(senseAdminMultisigAddress);
-        divider = divider.connect(multisigSigner);
-        periphery = periphery.connect(multisigSigner);
+      }
+
+      if ([CHAINS.HARDHAT, CHAINS.GOERLI].includes(chainId)) {
+        if (chainId === CHAINS.HARDHAT) {
```
Comment (abylaw) [inline pkg/deployments/tasks/20221031-factories-and-morpho/index.js:95]: nit: move this `if` outside of parent `if`

possibly extract into a helper as well - `fundMultisig`?

File: pkg/deployments/tasks/20221031-factories-and-morpho/index.js [hunk 3]
```diff
@@ -82,85 +82,93 @@ task("20221031-factories-and-morpho", "Deploys 4626 Factories & maUSDC & maUSDT"
...
+          const multisigSigner = await hre.ethers.getSigner(senseAdminMultisigAddress);
+          divider = divider.connect(multisigSigner);
+          periphery = periphery.connect(multisigSigner);
+        }
 
         console.log(`\n - Add ${factoryContractName} support to Periphery`);
         await (await periphery.setFactory(factoryAddress, true)).wait();
 
         console.log(`\n - Set factory as trusted address of Divider so it can call setGuard()`);
         await (await divider.setIsTrusted(factoryAddress, true)).wait();
```
Comment (abylaw) [inline pkg/deployments/tasks/20221031-factories-and-morpho/index.js:110]: remind me why we don't do the above two actions on mainnet again?

File: pkg/deployments/tasks/20221031-factories-and-morpho/index.js [hunk 4]
```diff
@@ -82,85 +82,93 @@ task("20221031-factories-and-morpho", "Deploys 4626 Factories & maUSDC & maUSDT"
...
+          console.log("\n-------------------------------------------------------");
+          console.log(`\nDeploy adapters for: ${factoryContractName}`);
+          console.log("\n-------------------------------------------------------");
 
-          // deploy Series
-          for (let m of t.series) {
-            console.log(`\n- Sponsor Series for ${t.name} @ ${t.address} with maturity ${m}`);
-            await periphery.callStatic.sponsorSeries(adapterAddress, m, false);
-            await (await periphery.sponsorSeries(adapterAddress, m, false)).wait();
+          for (const t of targets) {
```
Comment (abylaw) [inline pkg/deployments/tasks/20221031-factories-and-morpho/index.js:118]: can we extract parts of the code below into a helper?

**Review by abylaw — APPROVED** [review]
LGTM!

File: pkg/deployments/tasks/20221031-factories-and-morpho/index.js (hunk 3 above)
Comment (jparklev) [reply pkg/deployments/tasks/20221031-factories-and-morpho/index.js:110]: they require the sense multisig!

Comment (abylaw) [reply pkg/deployments/tasks/20221031-factories-and-morpho/index.js:110]: ahhhh! right.

**Pull request merged**